src/DDoSX/Entities/DomainProperty.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace UKFast\SDK\DDoSX\Entities;
+
+use UKFast\SDK\Entity;
+
+/**
+ * @property string $id
+ * @property string $name
+ * @property integer $value
```

Comment:
from what I've seen, $value can be bool, string or int - so I'd stick `mixed` here
```suggestion
 * @property mixed $value
```